### PR TITLE
Moving Stefan's talk to previous talks section

### PR DESCRIPTION
**Pull request by psocratis**

### CODE REVIEW

**Comment by tzanio** [comment]
Thanks @psocratis, can you add also the `Slides` and `Talk recording` buttons (you can add the slides and put an empty link for the video).

@kmittal2, do you have title/abstract for Som's talk that we can add in this PR?

**Comment by kmittal2** [comment]
> @kmittal2, do you have title/abstract for Som's talk that we can add in this PR?

Som will be sending it to me within the next day or so.

**Comment by tzanio** [comment]
> > @kmittal2, do you have title/abstract for Som's talk that we can add in this PR?
> 
> Som will be sending it to me within the next day or so.

Ok, I'll wait to add it in this PR

File: src/seminar.md [hunk 1]
```diff
@@ -12,26 +12,57 @@ Fill in [this form](https://docs.google.com/forms/d/e/1FAIpQLScrJ9QT7v7abx2ELcET
...
+The work presented here has been facilitated by funding through DOE’s National Virtual Biotechnology Laboratory (NVBL).
+
+---
+
+&nbsp;
+
+#### Som Dutta (Utah State University)
+##### **February 07, 2023**
 
 ---
```
Comment (tzanio) [inline src/seminar.md:41]: Duplicate?

**Review by tzanio — APPROVED** [review]
Thanks @psocratis and @kmittal2 !

**Pull request merged**